var/spack/repos/builtin/packages/root/package.py

```diff
@@ -50,71 +61,432 @@ class Root(CMakePackage):
         patch('math_uint.patch', when='@6.06.02')
         patch('root6-60606-mathmore.patch', when='@6.06.06')
 
-    variant('graphviz', default=False, description='Enable graphviz support')
+    # ###################### Variants ##########################
+
+    variant('avahi', default=False,
+        description='Compile with avahi')
+    variant('aqua', default=False,
+        description='Enable Aqua interface')
+    # variant('asimage', default=True,
+    #    description='Enable image processing support')
+    variant('davix', default=True,
+        description='Compile with external Davix')
+    variant('emacs', default=False,
+        description='Enable Emacs support')
+    variant('examples', default=True,
+        description='Install examples')
+    variant('fftw', default=False,
+        description='Enable Fast Fourier Transform support')
+    variant('fits', default=False,
+        description='Enable support for images and data from FITS files')
+    variant('fortran', default=False,
+        description='Enable the Fortran components of ROOT')
+    variant('graphviz', default=False,
+        description='Enable graphviz support')
+    variant('gdml', default=True,
+        description='Enable GDML writer and reader')
+    variant('gsl', default=True,
+        description='Enable linking against shared libraries for GSL')
+    variant('http', default=False,
+        description='Enable HTTP server support')
+    variant('jemalloc', default=False,
+        description='Enable using the jemalloc allocator')
+    variant('kerberos', default=False,
+        description='Enable Kerberos support')
+    variant('ldap', default=False,
+        description='Enable LDAP support')
+    variant('libcxx', default=False,
+        description='Build using libc++')
+    variant('math', default=True,
+        description='Build the new libMathMore extended math library')
+    variant('memstat', default=True,
+        description='Enable a memory stats utility to detect memory leaks')
+    # variant('minuit', default=True) - not supported by Spack
+    # variant('mysql', default=False) - not supported by spack
+    variant('odbc', default=False,
+        description='Enable ODBC support')
+    variant('opengl', default=True,
+        description='Enable OpenGL support')
+    # variant('oracle', default=False) - not supported by spack
+    variant('postgres', default=False,
+        description='Enable postgres support')
+    variant('pythia6', default=False,
+        description='Enable pythia6 support')
+    variant('pythia8', default=False,
+        description='Enable pythia8 support')
+    variant('python', default=True,
+        description='Enable Python ROOT bindings')
+    variant('qt4', default=False,
+        description='Enable Qt graphics backend')
+    variant('r', default=False,
+        description='Enable R ROOT bindings')
+    variant('rpath', default=False,
+        description='Enable RPATH')
```

Comment:
Considering Spack's general stance on RPATHs, this option should probably be on by default.